README.md

```diff
@@ -49,6 +49,28 @@ You can also find them on [the community calendar](https://calendar.google.com/c
 
 Meeting agendas and notes can be accessed in the [working doc](http://bit.ly/istiocommunitymeet).
 
+Interested in presenting? Please send your abstract to istio-steering@googlegroups.com.
+This is the information we need from you:
+    * Name
+    * Role and Company
+    * Short Speaker Bio
+    * Talk title and a short abstract
+    * Talk Format(Presentation/Demo) and Session Length
+    * Email to connect
+
+Suitable content:
+
+    * Istio new feature releases and project updates
+    * Tutorials and demos
+    * Use cases
+    * Content that relates vendor or platform to Istio installation and use
```

Comment:
It should be ok as long as it is open sourced, otherwise vendor private should not be encouraged or allowed

Reply:
That has been added as a point in the "Unsuitable Content" section

Reply:
I suggested a small change to make it clear